README.md

```diff
@@ -51,18 +74,28 @@ By default, `nef` can create an Xcode Playground with support for [Bow](http://b
 ➜ nef playground
 ```
 
+<p align="center">
+<img src="assets/nef-playground.png" height="100">
+</p>
+
 And you can use the following option to specify the name for the Xcode project that you are creating:
 
 ```bash
 ➜ nef playground --name LatestBowProject
 ```
 
-It will create an Xcode project with support for the latest available version of Bow, named `BowPlayground`. If you open this Xcode project, you will have an Xcode Playground where you can import Bow or any of its modules, and start trying some of its features.
+It will create an Xcode project with support for the latest available version of Bow, named `LatestBowProject`. If you open this `nef playground`, you will have an Xcode Playground where you can import Bow or any of its modules, and start trying some of its features.
+
+By default, `nef playground` will be created for iOS platform. If you need to change it, you can use the `--platform` option.
+
+```bash
+➜ nef playground --platform osx
+```
 
 ###
```

Comment:
What are these empty section titles?

Reply:
these are new lines, to add v-spacing between lines.